Cargo.toml

```diff
@@ -19,22 +19,46 @@ exclude = [
 [workspace]
 members = [
     "core",
-    "decode_wav",
+    "decode_symphonia",
     "encode_wav",
     "demos/player",
     "demos/writer",
 ]
 
 [features]
-default = ["decode-wav-only", "encode-wav"]
-decode-wav-only = ["creek-decode-wav"]
+default = ["decode", "encode-wav"]
+decode = [ "creek-decode-symphonia" ]
+decode-aac = [ "creek-decode-symphonia/aac" ]
+# Added after Symphonia 0.4 release
+# Be sure to add this to decode-all when Symphonia makes a new release
+#decode-alac =  [ "creek-decode-symphonia/alac" ]
+decode-flac = [ "creek-decode-symphonia/flac" ]
+decode-mp3 =  [ "creek-decode-symphonia/mp3" ]
+# https://github.com/pdeljanov/Symphonia/pull/81
+#decode-opus = [ "creek-decode-symphonia/opus" ]
+decode-pcm = [ "creek-decode-symphonia/pcm" ]
+decode-isomp4 = [ "creek-decode-symphonia/isomp4" ]
+# https://github.com/pdeljanov/Symphonia/pull/80
+#decode-mkv = [ "creek-decode-symphonia/mkv" ]
+decode-ogg = [ "creek-decode-symphonia/ogg" ]
+decode-vorbis = [ "creek-decode-symphonia/vorbis" ]
+decode-wav = [ "creek-decode-symphonia/wav" ]
 decode-open-source = []
-decode-all = []
+decode-all = [
+  "decode-aac",
+  "decode-flac",
+  "decode-mp3",
+  "decode-pcm",
+  "decode-isomp4",
+  "decode-ogg",
+  "decode-vorbis",
+  "decode-wav"
+]
 encode-wav = ["creek-encode-wav"]
 
 [dependencies]
 creek-core = { version = "0.1", path = "core" }
-creek-decode-wav = { version = "0.1.0", path = "decode_wav", optional = true }
+creek-decode-symphonia = { version = "0.1.0", path = "decode_symphonia", optional = true }
```

Comment:
Unrelated: I prefer to omit the patch version in library dependencies. Pinning the major/minor version should be sufficient.